server/chat-commands/info.ts

```diff
@@ -2569,11 +2572,16 @@ export const commands: ChatCommands = {
 			const [width, height, resized] = request.dimensions;
 			buf = Utils.html`<img src="${request.link}" width="${width}" height="${height}" />`;
 			if (resized) buf += Utils.html`<br /><a href="${request.link}" target="_blank">full-size image</a>`;
-		} else {
-			const YouTube = new YoutubeInterface();
+		}
+		const Spotify = new SpotifyInterface();
+		const YouTube = new YoutubeInterface();
+		if (Spotify.linkRegex.test(request.link)) {
+			buf = Spotify.generateSongDisplay(request.link);
+		} else if (YouTube.linkRegex.test(request.link)) {
 			buf = await YouTube.generateVideoDisplay(request.link);
 			if (!buf) return this.errorReply('Could not get YouTube video');
 		}
+		}
```

Comment:
```suggestion
```

Reply:
No need for suggestions, I am just removing SpotifyInterface class and doing what mia suggested.